app/src/main/java/com/github/ybecker/epforuml/authentication/FirebaseAuthenticator.kt

```diff
@@ -33,6 +33,7 @@ class FirebaseAuthenticator(
         FirebaseAuthUIActivityResultContract()
     ) { res -> this.onSignInResult(res) }
 
+    /*
     init {
```

Comment:
You need to decide whether to keep this code or remove this comment. 
I believe this code automatically connects you if your credentials are found. If you choose to keep it, you will likely face some issues with the profile picture (and maybe even other variables) as it is not added to the user. 
Please make sure to address this before merging!

Reply:
My bad ! Those lines are indeed not needed anymore as we now check and skip the login process in checkIfAlreadySignedIn in LoginActivity (there is also similar commented code in DatabaseManager that I will remove). This function is cleaner and we don't need to create a new user but just fetch it from the database.